source/locales.tex

```diff
@@ -4689,9 +4689,9 @@
     class InputIterator = istreambuf_iterator<charT> >
   class money_get : public locale::facet {
   public:
-    typedef charT               char_type;
-    typedef InputIterator       iter_type;
-    typedef basic_string<charT> string_type;
+    using   char_type = charT;
```

Comment:
Why right-align? Isn't that a bit cute? We used to left-align everything previously.